tests/test_instrument_method.py

```diff
@@ -41,17 +41,17 @@ def test_column_handler(self):
     def test_instrument_method(self):
         minimal_definition = {"name": "none_of_the_above"}
         instrument_method = instrument_method_factory(minimal_definition)
-        assert isinstance(instrument_method, InstrumentMethod)
+        assert isinstance(instrument_method, EmpowerInstrumentMethod)
         # This is a PDA, we do not have specific classes for detectors
```

Comment:
I have example samplesets for TUV if we wont to create one down the line

Reply:
I think we have one from Waters as well.